api/src/Entity/ScheduleEntry.php

```diff
@@ -270,4 +271,18 @@ public function getNumber(): string {
 
         return $dayNumber.'.'.$scheduleEntryStyledNumber;
     }
+
+    /**
+     * @param ScheduleEntry $prototype
+     * @param EntityMap     $entityMap
+     */
+    public function copyFromPrototype($prototype, &$entityMap = null) {
```

Comment:
Kann man nicht EntityMap als Typ angeben, und so auf die annotation verzichten?

Reply:
Und wenn die EntityMap eine Klasse ist, kann man dann nicht auf das & verzichten, da nur eine Referenz kopiert wird.
Oder willst du das als IN/OUT Parameter markieren?